packages/volto/src/components/manage/Blocks/LeadImage/Edit.jsx

```diff
@@ -23,124 +16,79 @@ const messages = defineMessages({
-   * @returns {undefined}
-   */
-  onAlignBlock(align) {
-    this.props.onChangeBlock(this.props.block, {
-      ...this.props.data,
-      align,
-    });
-  }
-
-  /**
-   * @param {*} nextProps
-   * @returns {boolean}
-   * @memberof Edit
-   */
-  shouldComponentUpdate(nextProps) {
-    return (
-      this.props.selected ||
-      nextProps.selected ||
-      !isEqual(this.props.data, nextProps.data)
-    );
-  }
+const Edit = React.memo((props) => {
+  const intl = useIntl();
+  const { data, properties, selected } = props;
 
-  node = React.createRef();
+  const placeholder = () =>
+    data.placeholder || intl.formatMessage(messages.ImageBlockInputPlaceholder);
 
-  /**
-   * Render method.
-   * @method render
-   * @returns {string} Markup for the component.
-   */
-  render() {
-    const Image = config.getComponent({ name: 'Image' }).component;
-    const { data, properties } = this.props;
-    const placeholder =
-      this.props.data.placeholder ||
-      this.props.intl.formatMessage(messages.ImageBlockInputPlaceholder);
+  const Image = config.getComponent({ name: 'Image' }).component;
+  const hasImage = !!properties.image;
+  const hasImageData = hasImage && !!properties.image.data;
+  const className = cx('responsive', { 'full-image': data.align === 'full' });
+  const altText = data.image_caption || properties.image_caption || '';
 
-    const hasImage = !!properties.image;
-    const hasImageData = hasImage && !!properties.image.data;
-    const className = cx('responsive', { 'full-image': data.align === 'full' });
-    const altText = data.image_caption || properties.image_caption || '';
-
-    return (
-      <div
-        className={cx(
-          'block image align',
-          {
-            center: !Boolean(data.align),
-          },
-          data.align,
-        )}
-      >
-        {!hasImage && (
-          <Message>
-            <center>
-              <img src={imageBlockSVG} alt="" />
-              <div className="message-text">{placeholder}</div>
-            </center>
-          </Message>
-        )}
-        {hasImage && hasImageData && (
-          <img
-            className={className}
-            src={`data:${properties.image['content-type']};base64,${properties.image.data}`}
-            width={properties.image.width}
-            height={properties.image.height}
-            alt={altText}
-            style={{
-              aspectRatio: `${properties.image.width}/${properties.image.height}`,
-            }}
-          />
-        )}
-        {hasImage && !hasImageData && (
-          <Image
-            className={className}
-            item={properties}
-            imageField="image"
-            sizes={(() => {
-              if (data.align === 'full' || data.align === 'center')
-                return '100vw';
-              if (data.align === 'left' || data.align === 'right')
-                return '50vw';
-              return undefined;
-            })()}
```

Comment:
The old code defined a function and called it immediately. We need to do the same thing in the new code. (This is why the snapshot changed, but it should not.)